src/test/java/com/shapesecurity/shift/integration/IntegrationTest.java

```diff
@@ -0,0 +1,89 @@
+package com.shapesecurity.shift.integration;
+
+import com.shapesecurity.functional.data.ImmutableList;
+import com.shapesecurity.shift.ast.Module;
+import com.shapesecurity.shift.ast.Node;
+import com.shapesecurity.shift.ast.Script;
+import com.shapesecurity.shift.codegen.CodeGen;
+import com.shapesecurity.shift.fuzzer.Fuzzer;
+import com.shapesecurity.shift.parser.JsError;
+import com.shapesecurity.shift.parser.Parser;
+import com.shapesecurity.shift.serialization.Serializer;
+import com.shapesecurity.shift.validator.ValidationError;
+import com.shapesecurity.shift.validator.Validator;
+import org.junit.Test;
+
+import java.util.Random;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertTrue;
+
+public class IntegrationTest {
+  @Test
+  public void testFuzzerToValidator() {
+    testFuzzerToValidatorHelper((long) 0, 5);
```

Comment:
These fixed tests are great, but let's also add a test that runs it for 5 to 10 seconds with random seeds. Make sure that the seed is printed out on error.